Products/DataCollector/zendisc.py

```diff
@@ -285,7 +286,7 @@ def findRemoteDeviceInfo(self, ip, devicePath, deviceSnmpCommunities=None):
             self.log.debug("Override acquired community strings")
             # Override the device class communities with the ones set on
             # this device, if they exist
-            communities = snmp_conf['zSnmpCommunities']
+            communities = snmp_conf['zSnmpCommunities'] or DEFAULT_COMMUNITIES
```

Comment:
Main goal of `or DEFAULT_COMMUNITIES` is to not allow to `community` variable contain blank value. Actually I don't care which value is set to `zSnmpCommunities` unless it's blank. ZEN-21177 issue exists because of `zSnmpCommunities == None`